xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserProfileSheet.xml

```diff
@@ -141,6 +141,27 @@
       <code>require(['jquery', 'xwiki-meta'], function ($, xm) {
   'use strict';
   $(document).ready(function() {
+
+    /** Add autocomplete attributes to ease form filling. */
+    const prefix = 'XWiki.XWikiUsers_0_';
```

Comment:
Globally not a big fan of hardcoded stuff like that... my first move would be "what if the page has several user xobjects?" but I guess in that case XWiki only take into account the first one for the sheet?

Reply:
Actually, having several objects is not really the main risk, if you delete the XWikiUsers object and add a new one the first object number will be 1 and not 0.

Reply:
Right, that was a very poor idea, thanks for the review. The latest commit uses instead a jQuery selector based on the input names ending, without any reference to the object(s) identifier, what do you think?